tools/ci_build/github/azure-pipelines/win-eager-ci-pipeline.yml

```diff
@@ -0,0 +1,132 @@
+parameters:
```

Comment:
Currently win-ci-pipeline.yml has three variations in the config matrix: x64_debug, x64_release  and x86_release. You can move the body of the build jobs to a template file and reuse them in these three and yours.  

And if you'd like, you can put your things in win-ci-pipeline.yml without the need of creating a new pipeline and new yaml file if you are confident on the stability.